core/includes/theme.inc

```diff
@@ -1940,6 +1940,13 @@ function template_preprocess_image(&$variables) {
   $path_or_uri = $variables['uri'] ? $variables['uri'] : $variables['path'];
   $variables['attributes']['src'] = file_create_url($path_or_uri);
 
+  if (isset($variables['timestamp'])) {
+    $variables['attributes']['src'] = $variables['attributes']['src'] . (strpos($variables['attributes']['src'], '?') !== FALSE ? '&' : '?') . backdrop_http_build_query(array('timestamp' => $variables['timestamp']));
+  }
+  else {
+    $variables['attributes']['src'] = file_create_url($variables['uri']);
```

Comment:
This fails for images provided via path (from modules or core), hence feed icon is broken (and many other images).

Using `file_create_url($path_or_uri);` fixes it. And fixes at least some of the broken tests.

Reply:
Thanks @indigoxela 👍 ...one step closer 🙂